client/blocks/reader-full-post/index.jsx

```diff
@@ -77,6 +76,11 @@ export class FullPostView extends React.Component {
 
 	hasScrolledToCommentAnchor = false;
 
+	constructor( props ) {
+		super( props );
+		this.commentsWrapper = React.createRef();
```

Comment:
don't really need the ctor here. You can use the property assignment syntax to get the same effect:

`commentsWrapper = React.createRef();`

Reply:
That's cleaner. Thanks @blowery!